src/pretix/base/models/auth.py

```diff
@@ -130,6 +132,8 @@ class User(AbstractBaseUser, PermissionsMixin, LoggingMixin):
                                    verbose_name=_('Is site admin'))
     date_joined = models.DateTimeField(auto_now_add=True,
                                        verbose_name=_('Date joined'))
+    needs_password_change = models.BooleanField(default=False,
+                                                verbose_name=_('Needs to change password on login'))
```

Comment:
Technically, we don't ask "on login", so maybe

```suggestion
                                                verbose_name=_('Force user to select a new password'))
```

would be more correct? This is really nitpicky, though, and I don't really care about it